packages/graph/hash_graph/lib/graph/src/shared/identifier/knowledge.rs

```diff
@@ -83,22 +82,21 @@ impl ToSchema for EntityId {
     }
 }
 
-#[derive(
-    Debug, Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize, FromSql, ToSql,
-)]
-#[repr(transparent)]
-#[postgres(transparent)]
-pub struct EntityVersion(Timestamp);
-
-impl EntityVersion {
-    #[must_use]
-    pub const fn new(inner: Timestamp) -> Self {
-        Self(inner)
-    }
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq, Deserialize)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct EntityVersion {
+    decision_time: TimeSpan,
+    system_time: TimeSpan,
+}
 
-    #[must_use]
-    pub const fn inner(&self) -> Timestamp {
-        self.0
+impl Serialize for EntityVersion {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        // TODO: Expose temporal versions to backend
```

Comment:
This is just to have a non-breaking change I assume?

Reply:
Yes, exactly